app/controllers/events_controller.rb

```diff
@@ -4,6 +4,14 @@ class EventsController < InheritedResources::Base
   skip_load_and_authorize_resource only: [:index, :show]
   before_filter :set_types
 
+  def participants
```

Comment:
Yes, why not. Or, if the PR get's to complex we could refactor this once this got merged.